static/src/javascripts/components/bean/bean.js

```diff
@@ -106,7 +106,7 @@
               'fromElement offsetX offsetY pageX pageY screenX screenY toElement'))
           , mouseWheelProps = mouseProps.concat(str2arr('wheelDelta wheelDeltaX wheelDeltaY wheelDeltaZ ' +
               'axis')) // 'axis' is FF specific
-          , keyProps     = commonProps.concat(str2arr('char charCode key keyCode keyIdentifier '          +
+          , keyProps     = commonProps.concat(str2arr('char charCode key keyCode '          +
```

Comment:
Yes, I think until the (issue is fixed in Bean)[https://github.com/fat/bean/issues/128] this is the only thing we can do. Can you add a comment to this PR? 👍